src/pages/zoeken/[slug]/[ie]/index.tsx

```diff
@@ -1319,17 +1320,46 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, url }) => {
 		</>
 	);
 
+	// To determine the correct error page or the object detail page, we follow this flow:
+	// fetch object
+	//    - 200: show detail page
+	//    - 404, 403: check the visitor space info
+	//           - 404: not found error page
+	//           - 410: visitor space no longer available error page
+	//           - 200: check visit request info
+	//                   - 200: not found error page
+	//                   - 404: no access error page
+
 	const renderPageContent = () => {
 		if (mediaInfoIsLoading || visitRequestIsLoading || visitorSpaceIsLoading) {
 			return <Loading fullscreen owner="object detail page: render page content" />;
 		}
-		if (isErrorSpaceNotActive) {
-			return <ErrorSpaceNoLongerActive />;
+		if (mediaInfo) {
+			return <div className="p-object-detail">{renderObjectDetail()}</div>;
 		}
-		if (isErrorNotFound) {
-			return <ErrorNotFound />;
+		if (isMediaInfoErrorNoAccess || isMediaInfoErrorNotFound) {
+			if (isErrorSpaceNotFound) {
+				return <ErrorNotFound />;
+			}
+			if (isErrorSpaceNotActive) {
+				return <ErrorSpaceNoLongerActive />;
```

Comment:
i would expect this case to be shown when i go to:
http://localhost:3200/zoeken/bruzzel/d7d7841f8ad946f8bfa5116b30b9041ccec8fc5521d2404cab08b86c2d41466e231c382d924942de8d586b2f713e85f6

the object id does not exist
and the visitor space has status: inactive

I think this is because the meemoo admin can still fetch inactive spaces, since they need to be able to change their status

So we'll have to change the logic a little bit:
```
if (isErrorSpaceNotActive || visitorSpace?.status === VisitorSpaceStatus.Inactive) {
```